src/main/java/org/cactoos/iterator/Mapped.java

```diff
@@ -53,7 +53,7 @@
      * @param func Func
      * @param iterator Source iterator
      */
-    public Mapped(final Func<X, Y> func, final Iterator<X> iterator) {
+    public Mapped(final Func<X, Y> func, final Iterator<? extends X> iterator) {
```

Comment:
@andreoss here too let's apply the same thing with super and extends on the func

Reply:
@victornoel Done